README.md

```diff
@@ -35,21 +35,21 @@ const result = await evervault.run('hello-cage', encrypted);
 
 ## Understanding the Cage
 This Cage is very simple. Here is the full code:
+
 ```javascript
-exports.handler = async (event) => {
-  console.debug('This is the data that has arrived into the Cage.', event);
+exports.handler = async (data) => {
+  if (data.name && data.name.length > 0) {
```

Comment:
Just have to paste this code block again with the latest updates

Reply:
oops